CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to
 
 ## [Unreleased]
 
+- cosmwasm-std: Export `GovMsg` as a top-level type of the crate.
```

Comment:
Could you create a `## [0.15.1] - 2021-07-20` section with a `### Fixed` subsection here? And the diff link for `[0.15.1]` in the bottom.